src/CompletedTaskList.js

```diff
@@ -2,12 +2,46 @@ import React from "react"
 
 import CreatedTask from "./CreatedTask"
 import Header from "./Header"
+import todosData from "./todosData"
 
-export default function CompletedTaskList(props) {
-  return (
-    <div>
-      <Header title="Completed Tasks" />
-      <CreatedTask task={props.task} completed="completed" />
-    </div>
-  )
-}
\ No newline at end of file
+class CompletedTaskList extends React.Component {
+  constructor() {
```

Comment:
We don't actually need to use a constructor anymore! Using the [alternative class syntax](https://www.robinwieruch.de/react-state-without-constructor) we can initialize state in the class itself!
```suggestion
 this.state = {
   todos: todosData
 }
```